packages/opentelemetry-node/docs/get-started.asciidoc

```diff
@@ -0,0 +1,209 @@
+be defined _before_ the {distro_name} is defined.
+====
+
+////
+✅ Start-to-finish operation
+////
+[discrete]
+[[get-started-send-data]]
+=== Send data to Elastic
+
+After installing the distro, configure and initialize it to start
+sending data to Elastic.
+
+////
+Are there any dependencies that need to be installed in addition to the distro?
+
+I think the answer here is no because the dependencies are already defined in the
+distro package so the user shouldn't have to individually install any other packages.
+If that's true, delete this comment.
+
+[discrete]
+[[get-started-add-dependencies]]
+==== Add dependencies
+////
+
+////
+✅ Provide _minimal_ configuration/setup
+////
+[discrete]
+[[get-started-configure]]
+==== Configure the distro
+
+////
+✅ Step-by-step instructions
+////
+To configure the distro, at a minimum you'll need the deployment's OTLP endpoint and
+authorization data to set the appropriate `OTLP_*` environment variables:
+
+include::configure.asciidoc[tag=minimal-options]
+
+When sending data to Elastic, you can find the values of these variables in Kibana's APM tutorial.
+In Kibana:
+
+. Search for _APM Tutorial_
+. Scroll down to the _APM Agents_ section and select the *OpenTelemetry* tab
+. The appropriate values for `OTEL_EXPORTER_OTLP_ENDPOINT` and
+`OTEL_EXPORTER_OTLP_HEADERS` are shown there
++
+For example:
++
+[source,sh]
+----
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://my-deployment.apm.us-west1.gcp.cloud.es.io
+export OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer P....l"
+----
+
+image::img/otlp-endpoint-settings.png[Kibana's APM tutorial showing OTel settings]
+
+For more information on all the available configuration options, refer to <<configure>>.
+
+////
+✅ Start sending data to Elastic
+////
+[discrete]
+[[get-started-initialization]]
+==== Initialize the distro
+
+For the distro to automatically instrument modules used by your Node.js service,
+it must be started before you `require` your service code's dependencies --
+for example, before `express` or `http` are loaded.
+
+////
+✅ Step-by-step instructions
+////
+The recommended way to get the
+distro started is by using the `-r, --require` Node.js
+https://nodejs.org/api/cli.html#-r---require-module[CLI option]:
+
+[source,sh]
+----
+node --require @elastic/opentelemetry-node my-service.js
+----
+
+The distro will automatically instrument popular modules (listed in <<supported-technologies>>)
+used by your service, and send traces, metrics, and logs telemetry data (using
+OTLP) to your configured observability backend.
+
+// TODO: link to a reference section on other ways to start the distro once we have those docs.
+
+////
+TO DO:
+What should they expect to see after setting up and running the distro?
+How do they know it worked as expected?
+////
```

Comment:
*Typically* I think we could say that they should see their service in the "Services" tab in Kibana.  However, I think that may require there to be some actual traces sent... which won't be the case if they haven't actually used their instrumented app in the last 15 minutes, so using that as  the "is this thing on?" test may invite some small confusion.

Playing a little bit, I ran this:

```
$ OTEL_SERVICE_NAME=eg1 node -r @elastic/opentelemetry-node http-server.js
{"name":"elastic-otel-node","level":30,"preamble":true,"distroVersion":"0.2.0","env":{"os":"darwin 23.5.0","arch":"arm64","runtime":"Node.js v18.20.4"},"msg":"start Elastic Distribution for OpenTelemetry Node.js","time":"2024-07-17T22:52:54.962Z"}
```

This is a simple Node.js service that just starts an HTTP server. I haven't make any HTTP requests to it, so the service hasn't had any traces to report. However, it *did* (after a couple minutes), show up in the Services list of the APM app in Kibana:

![Screenshot 2024-07-17 at 3 55 26 PM](https://github.com/user-attachments/assets/1c00101e-edf8-43f8-aa36-967884cd1ced)

That entry will be because, while no *trace* data was sent, the distro is periodically sending some basic metrics.

---

Another thing to confirm that it is working is that the distro will log a `start Elastic Distribution for OpenTelemetry Node.js` log message at startup. Assuming the user is able to see the stdout from their service (this might not be the case in some deployments), then that is an indication that the distro is up and running.  This is logged at the "INFO" level. If the user sets `OTEL_LOG_LEVEL=warn`  (or 'error' or 'none'), to only see higher priority log output, then of course they won't see it.